src/test/enhetstester/kotlin/no/nav/familie/ba/sak/internal/GithubControllerTest.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.familie.ba.sak.internal
+
+import no.nav.familie.ba.sak.config.GithubConfig
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class GithubControllerTest {
+    private val githubConfig: GithubConfig = GithubConfig(githubBranch = "tester", githubSha = "123")
+    private val githubController = GithubController(githubConfig = githubConfig)
```

Comment:
Er kjekt å være konsekvent her. Tror kanskje øverste forslag er det som er brukt mest i andre tester. 

```suggestion
    private val githubConfig: GithubConfig = GithubConfig(githubBranch = "tester", githubSha = "123")
    private val githubController:  GithubController  = GithubController(githubConfig = githubConfig)
```

eller

```suggestion
    private val githubConfig = GithubConfig(githubBranch = "tester", githubSha = "123")
    private val githubController = GithubController(githubConfig = githubConfig)
```